datacenter/ucp/2.2/guides/admin/install/system-requirements.md

```diff
@@ -9,23 +9,34 @@ Before installing, be sure your infrastructure has these requirements.
 
 ## Hardware and software requirements
 
-You can install UCP on-premises or on a cloud provider. To install UCP,
-all nodes must have:
+You can install UCP on-premises or on a cloud provider. Common requirements:
 
 * [Docker Enterprise Edition](/engine/installation/index.md) version 17.06 or higher
 * Linux kernel version 3.10 or higher
+* A static IP address
+
+### Minimum requirements
+
 * 8GB of RAM for manager nodes or nodes running DTR
 * 4GB of RAM for worker nodes
 * 3GB of free disk space
-* A static IP address
+
+### Recommended production requirements
+
+ * 16GB of RAM for manager nodes or nodes running DTR
+ * 4 vCPUs for manager nodes or nodes running DTR
+ * 25-100GB of free disk space
+ 
+Note that Windows container images are typically larger than Linux ones and for that reason, you should consider provisioning more local storage for Windows nodes and for DTR setups that will store Windows container images.
```

Comment:
@friism does it make sense to also mention that admins should configure Docker EE to store logs using an external system? By default I think that the Daemon doesn't delete logs, which means that in a production system with intense usage, your logs will also take a significant amount of disk space.

Reply:
@joaofnfernandes I think that's a worthy recommendation, but I also want to keep this doc tight and focused on system requirements. Otherwise it'll just morph into a kitchen-sink guide with assorted random recommendations.